jp2_online/static/vendors/Chart.js/package.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
This is Chart.js should not be manually installed in this project, it should be handled through npm, so we can safely ignore it from the repository, it only adds a ton of fluff, and completely superfluous code to the repository.

We don't need the ddocumentation, the samples or even the package itself to be part of our code, specially for components that are available through npm.

http://www.chartjs.org/docs/